ansible/roles/common/templates/conf/output/01-es.conf.j2

```diff
@@ -1,32 +1,46 @@
+{% if enable_caso | bool and inventory_hostname in groups['caso'] %}
+<match apel.events>
+    @type copy
+    <store>
+       @type opensearch
+       host { opensearch_address }}
+       port {{ opensearch_port }}
+       logstash_format true
+       logstash_prefix apel
+       flush_interval 15s
+    </store>
+</match>
+{% endif %}
+
 <match **>
     @type copy
     <store>
-       @type elasticsearch
```

Comment:
I don't think these changes are necessary - there's another file for OpenSearch - 03-opensearch.conf.j2.

It does make me think that the apel.events bit should be in 03-opensearch.conf.j2 though.